pkg/gardener/shoot/extender/auditlogs/auditlogs_configuration.go

```diff
@@ -0,0 +1,29 @@
+package auditlogs
+
+import "fmt"
+
+var (
+	ErrConfigurationNotFound = fmt.Errorf("audit logs configuration not found")
+	zero                     = AuditLogData{}
+)
+
+type AuditlogsConfiguration map[string]map[string]AuditLogData
```

Comment:
Same type is already declared in main.go as

`type auditlogDataMap = map[provider]map[region]auditlogs.AuditLogData`